notebooks/build/covid-19.ipynb

```diff
@@ -0,0 +1,4219 @@
+{
```

Comment:
<p>This looks pretty crazy -- would adding some fuzz to the Columbia predictions help?</p><p> </p><p>The other thing we could try to do is to reconstruct the full COlumbia distributions, since we have the percentiles -- that might be worth playing around with</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oughtinc/ergo/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>